### PR TITLE
refactor: modify args to subcommand in dtm develop

### PR DESCRIPTION
**Pull request by xavier-hou**
## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [ ] I have read through the [CONTRIBUTING.md](../CONTRIBUTING.md) documentation.
- [ ] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

## Description

Modify `create-plugin` and `validate-plugin` from `developAction`(arguments) to subcommand.

## Related Issues

#493 

## New Behavior (screenshots if needed)

- `dtm develop`
image

- `dtm develop create-plugin`
image

- `dtm develop validate-plugin`
image

### CODE REVIEW

**Comment by hxcGit** [comment]
Further consideration is needed.

File: cmd/devstream/develop.go [hunk 1]
```diff
@@ -19,31 +15,44 @@ var (
...
+
+var developValidateCMD = &cobra.Command{
+	Use:   "validate-plugin",
+	Short: "Validate a plugin",
+	Long: `Validate-plugin is used for validating an existing plugin or all plugins.
 Examples:
-  dtm develop create-plugin --name=YOUR-PLUGIN-NAME,
-  dtm develop validate-plugin --name=YOUR-PLUGIN-NAME`,
-	Run: options.WithValidators(developCMDFunc, options.ArgsCountEqual(1), validateDevelopArgs),
+	dtm develop validate-plugin --name=YOUR-PLUGIN-NAME,
```
Comment (daniel-hutao) [inline cmd/devstream/develop.go:34]: validate-plugin -> create-plugin

File: cmd/devstream/develop.go [hunk 2]
```diff
@@ -19,31 +15,44 @@ var (
...
+	dtm develop validate-plugin --all`,
```
Comment (daniel-hutao) [inline cmd/devstream/develop.go:35]:
```sh
dtm develop validate-plugin --all
```

to 

```sh
dtm develop validate-plugin --name=YOUR-PLUGIN-NAME
dtm develop validate-plugin --all
```

File: cmd/devstream/develop.go [hunk 3]
```diff
@@ -19,31 +15,44 @@ var (
 var developCMD = &cobra.Command{
 	Use:   "develop",
 	Short: "Develop is used for develop a new plugin",
-	Long: `Develop is used for develop a new plugin.
+}
+
+var developCreateCMD = &cobra.Command{
```
Comment (daniel-hutao) [inline cmd/devstream/develop.go:20]: Maybe it's better to use `createPluginCMD` or `developCreatePluginCMD`

File: cmd/devstream/develop.go [hunk 4]
```diff
@@ -19,31 +15,44 @@ var (
...
+	Use:   "create-plugin",
+	Short: "Create a new plugin",
+	Long: `Create-plugin is used for creating a new plugin.
+Exampls:
+	dtm develop create-plugin --name=YOUR-PLUGIN-NAME`,
+	Run: developCreateCMDFunc,
+}
+
+var developValidateCMD = &cobra.Command{
```
Comment (daniel-hutao) [inline cmd/devstream/develop.go:29]: Maybe it's better to use `validatePluginCMD` or `developValidatePluginCMD`

**Review by daniel-hutao — CHANGES_REQUESTED** [review]
See the comments

File: cmd/devstream/develop.go (hunk 1 above)
Comment (hxcGit) [reply cmd/devstream/develop.go:34]: Here I put the example information into two different subcommands.

File: cmd/devstream/develop.go (hunk 2 above)
Comment (hxcGit) [reply cmd/devstream/develop.go:35]: I don't quite understand your meaning.

File: cmd/devstream/develop.go (hunk 1 above)
Comment (daniel-hutao) [reply cmd/devstream/develop.go:34]: Sorry, my mistake.

File: cmd/devstream/develop.go (hunk 2 above)
Comment (daniel-hutao) [reply cmd/devstream/develop.go:35]: Sorry, my mistake.

**Pull request merged**